src/main/java/storagesystem/model/StoreIT.java

```diff
@@ -17,18 +19,43 @@
     /**
      * Loads all data into the program. Should be run at start.
      */
-    public void initializeBackend() {
-        mockData();
+    public void initializeBackend() throws IOException {
+        //JSONHandler.clearAllJsonFiles(); //Run with mockData() if fresh start or after tests!!!
+        //mockData();
+
+        try {
+            organisations.addAll(JSONHandler.getListFromJson(Organisation.class));
+        } catch (NullPointerException e) {
+            System.out.println("Organisation json is empty.");
+        }
+        System.out.println("Loaded Organisations from json.");
+        IDHandler.updateAllIDs(organisations);
+        currentOrganisation = organisations.get(0);
+        System.out.println("Current Organisation Set.");
     }
 
-    private static void mockData() {
+    private static void mockData() throws IOException {
```

Comment:
maybe just clear all json files when this method is run?

Reply:
added reset() method instead